packages/teams_memory/teams_memory/storage/in_memory_storage.py

```diff
@@ -318,7 +318,12 @@ async def get_conversations_from_buffered_messages(
         ref_dict: Dict[str, List[str]] = {}
         for key, value in self.storage["buffered_messages"].items():
             stored_message_ids = [item.id for item in value]
-            common_message_ids: List[str] = np.intersect1d(np.array(message_ids), np.array(stored_message_ids)).tolist()  # type: ignore
+            common_message_ids = [
+                str(x)
```

Comment:
NIT: isn't message_id a str? why do we need to cast it?

Reply:
Needed because `original_send_activities` from BF doesn't have types.